jekyll/_cci2/scheduled-pipelines.md

```diff
@@ -36,7 +36,62 @@ You can find a basic how-to guide on the [Set a nightly scheduled pipeline](/doc
 ## Get started with scheduled pipelines
 {: #get-started-with-scheduled-pipelines }
 
-To get started with scheduled pipelines, you have the option of using the API, or using the CircleCI web app. Both methods are described below.
+To get started with scheduled pipelines, you have the option of using the Scheduled Pipelines Orb, the API, or using the CircleCI web app. Each method is described below.
+
+### Use Scheduled Pipelines Orb
```

Comment:
Recommend positioning the orb under the web app, which I imagine is the "main" interaction method.